lib/MetaCPAN/Script/Release.pm

```diff
@@ -490,6 +498,73 @@ sub import_tarball {
     }
 }
 
+sub find_main_module {
+    my $self = shift;
+    my ( $mod, $distribution ) = @_;
+
+    my @modules = @{$mod};
+
+    my $min_level          = 100;
+    my @distribution_parts = split /-/, $distribution;
+    my $distribution_name  = join '', @distribution_parts;
+
+    if ( scalar @modules eq 1 ) {
```

Comment:
`if ( scalar @modules == 1 ) {`

You want a numeric comparison (not string).  Even if it actually works, using the `==` is less confusing.